# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/spirv_headers/.cargo-checksum.json
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/spirv_headers/.cargo-checksum.json b/third_party/rust/spirv_headers/.cargo-checksum.json
deleted file mode 100644
--- a/third_party/rust/spirv_headers/.cargo-checksum.json
+++ /dev/null
@@ -1,1 +0,0 @@
-{"files":{"Cargo.toml":"28afa2776d1c255a806014b431de88f8ab382fee3363e77dc7dc181040298f62","README.md":"178e8c582515c3565ecb354e43e59d8f78ae2a48aa298b1bb7e2241614af3d9f","autogen_spirv.rs":"5b983427fcf8a4fa73e291f76c83b27627ef8464d39c5bbf4312158937bfb986","lib.rs":"463c4649b1bc16a9823ec48029aaa70ae06fc3028682cc9573cc8565bd9f7ba5"},"package":"1f5b132530b1ac069df335577e3581765995cba5a13995cdbbdbc8fb057c532c"}
\ No newline at end of file